decidim-system/app/controllers/decidim/system/application_controller.rb

```diff
@@ -9,6 +9,18 @@ class ApplicationController < ActionController::Base
       helper Decidim::TranslationsHelper
       helper Decidim::DecidimFormHelper
       helper Decidim::ReplaceButtonsHelper
+
+      def append_info_to_payload(payload)
```

Comment:
Why not move it to `Decidim::ApplicationController`? I think everything inherits from this one.